dotcom-rendering/src/components/AppEmailSignUp.tsx

```diff
@@ -17,22 +17,18 @@ export const AppEmailSignUp = ({
 	skipToIndex,
 	...emailSignUpProps
 }: AppEmailSignupProps) => {
-	const isCompatible = useIsBridgetCompatible();
```

Comment:
This check was carried over from Apps Rendering. The apps teams have confirmed that this check is outdated and no longer required.